.buildkite/pipeline.yml

```diff
@@ -0,0 +1,19 @@
+agents:
+  queue: "public"
+
+steps:
+- name: "Go build"
+  command: "go build -v -x -buildmode=c-shared -o planetscale-linux.so.$BUILDKITE_COMMIT && buildkite-agent artifact upload planetscale-linux.so.$BUILDKITE_COMMIT"
+  plugins:
+    - docker#v3.8.0:
+        image: "golang:1.16.4"
```

Comment:
I copied this image from https://github.com/planetscale/planetscale-go/blob/6561257b3d4c43eb4cff32cd1d58c90c1c5b26bd/docker-compose.yml#L5.
Unclear about how worth it is to keep both images pinned to patch release.